lite/environment.yml

```diff
@@ -0,0 +1,19 @@
+name: xeus-python-kernel
```

Comment:
Arf, It'd be great to not duplicate this with what is in the docs.

Can you try removing those files (`environment.yml`, `jupyter-lite.json`, `jupyter_lite_config.json`) from the docs?

And instead change [this line](https://github.com/geojupyter/jupytergis/blob/main/docs/conf.py#L45) to being `../lite` ?

Reply:
Sure!

Reply:
As discussed privately, it's not easy to dedupe now. Let's keep track of this in an issue.